cassandra_read.js

```diff
@@ -0,0 +1,21 @@
+const cassandra = require('cassandra-driver');
+
+let client = new cassandra.Client({
+  contactPoints: ['localhost'],
+  localDataCenter: 'datacenter1',
+  keyspace: 'sdc_kaz',
+  });
+
+
+query = "select * from albums where id=10000000"
+
+startTime = (new Date()).getTime()
+client.execute(query, function(err, result) {
+  console.log(result.first())
+  if (err) {
+    console.log(err);
+  } else {
+    endTime = (new Date()).getTime()
+    console.log((endTime-startTime) + " ms")
```

Comment:
Nice use of the Date object for time tracking! I think it's cool you handled this natively instead of using a library.